training.md

```diff
@@ -36,45 +36,123 @@ There may be an issue where your terminal can't find Git or Anaconda, in which c
+   * `python extension`
+   * `better jinja`
+   * `vscode-dbt`
+   * `rainbow csv`
+3. Now we need to set up `profiles.yml` to connect to our database. In our repo, this is called [`profiles_example.yml`](profiles_example.yml).
+   1. Create new directory called `.dbt` under your user folder (`C:\Users\your_user_folder`).
+   2. Open `profiles_example.yml`.
+   3. Save As the file into the `.dbt` directory you just made and rename it to `profiles.yml`.
+   4. Edit the file in VSCode and change the `schema` field on line 5 to your initials or anything that serves as an identifier for your work.
+4. Create your conda environment and install the necessary packages in here **once** so that you can reuse this environment for future dbt work.
+   1. Open a new terminal by clicking on "Terminal" at the top and "New Terminal"
+   2. Click "Allow" when a pop up appears asking if you allow the workspace to modify your terminal shell.
+      * If you see "PS" next to your current directory on the command line, that means you're using powershell and we don't want to use that. The fix: open a new terminal again and you should now be using cmd.
+   3. Since you already installed Anaconda, you should see `conda activate base` run automatically and (base) should be next to your current directory.
+   4. Create a new, empty environment by running `conda create -n dbtenv python=3.7.9`. The name of your environment is `dbtenv` and we are using Python version 3.7.9.
+      * *do we want to include this or should we just have them install in the base environment?*
+   5. Install packages
+      1. Run `pip install dbt-sqlserver`
+      2. Run `pip install azure-cli`
+5. Log into Azure. We want to connect to the database via Azure CLI.
+   1. Run `az login`
+   2. In the window that popped up, sign in with your Avanade credentials. Once signed, in you can close out of that window.
+   3. If you belong to multiple subscriptions, you must specify the subscription by running `az account set --subscription ff2e23ae-7d7c-4cbd-99b8-116bb94dca6e`. This is the ID for AzureCloud.
+6. Let's verify that we can connect to the database successfully.
+   1. Run `dbt debug`. This command tries to connect to the database using the parameters from `profiles.yml` and `dbt_project.yml`.
+      * You connected successfully if you see all green and no error messages!
+
+### Azure Data Studio
+
+1. Last step in the set-up is to log in to the database
+   1. Open Azure Data Studio and click on the first icon on the left side panel called "Connections".
+   2. Click "Add Connection" and paste in the connection info below in the relevant fields. This info can also be found in `profiles.yml`
+      * Server: dbtavatrain.database.windows.net
+      * Authentication type: Azure Active Directory
+      * Account: Click the drop down menu, then "Add an account" and sign in like before
+      * Azure AD tenant: Avanade
```

Comment:
Is this still relevant for the people you'll be speaking to this week?

Reply:
nope! they'll have to use their own database and database credentials. In a future PR, we should make a tutorials for how to do that with Azure SQL